data_structures/singly_linked_list.py

```diff
@@ -1,18 +1,12 @@
 class Node:
-  
-  value = None
-  succ = None
-
```

Comment:
Thanks! Also another useful comment from you: an instance field will shadow the class field which share the same name with it.